satellite/console/consoleweb/server.go

```diff
@@ -62,12 +62,13 @@ type Config struct {
 
 	PasswordCost int `internal:"true" help:"password hashing cost (0=automatic)" default:"0"`
 
+	ContactInfoURL        string `help:"url link to contacts page" default:"https://forum.storj.io"`
+	FrameAncestors        string `help:"allow domains to embed the satellite in a frame" default:"tardigrade.io"`
```

Comment:
Is it worth mentioning in the help text (or in a comment maybe) that multiple domains should be space-separated? I always hate having to go spelunking through code to figure out what an app expects

Reply:
I'm always up for better documentation.